tasks/RedHat/generic/configure-uuidd.yml

```diff
@@ -4,16 +4,16 @@
 - debug:
     msg: "imported RedHat/generic/configure-uuidd.yml"
 
-- name: Start and enable service uuidd
+- name: Enable and restart service uuidd
   systemd:
     name: uuidd
-    state: started
+    state: restarted
```

Comment:
using `restarted` here makes this task not idempotent - it will report changed every time

Reply:
Until last week, I used "state: started". But then I discovered a case where uuidd and uuidd.socket were not running after the role had been run. There was an error message about not enough free sockets or something similar. Restarting uuidd and uuidd.socket solved the issue, so I decided to always restart the two. I could however check if the services are running after using "state: started" and only restart if they are not running.

Reply:
That sounds like a bug in in the `systemd` module or perhaps in systemd itself.  I would prefer to investigate the root cause of this problem and fix it rather than putting in a workaround in ansible that breaks idempotency.

Reply:
OK, that makes sense. I'll go back to the previous code (state: started) again and we'll try to reproduce the issue during QA. The previous code has never been reported to be an issue, probable because the role is typically not executed more than 10 times or so in a short time, as during my development and testing.

Reply:
solved in https://github.com/berndfinger/sap-preconfigure/commit/bd0edabe4d62da57af2ca027dd0482e45f5ae083 .

Reply:
> probable because the role is typically not executed more than 10 times or so in a short time, as during my development and testing

I've seen something similar testing on Fedora when services are restarted several times in a short period of time - I had to add
```
[Unit]
StartLimitIntervalSec=0
```
to the systemd `.service` file for those services so that systemd would not give an error.

Reply:
e.g. you might have to the same to `/usr/lib/systemd/system/uuidd.service` in your test code.